pkg/util/yaml_test.go

```diff
@@ -233,3 +234,25 @@ func TestToYAMLWithJSONPB(t *testing.T) {
 		})
 	}
 }
+
+func TestUnmarshalWithJSONPB(t *testing.T) {
+	unmarshalWithJSONPBTests := []struct {
+		desc string
+		yaml string
+		want *v1alpha2.IstioControlPlaneSpec
+	}{
+		{"UnmarshalWithJSONPBToYAML", icpYaml, icp},
+	}
+
+	for _, tt := range unmarshalWithJSONPBTests {
+		t.Run(tt.desc, func(t *testing.T) {
+			got := &v1alpha2.IstioControlPlaneSpec{}
+			err := UnmarshalWithJSONPB(icpYaml, got)
+			if err != nil {
+				if !reflect.DeepEqual(got, tt.want) {
+					t.Errorf("TestUnmarshalWithJSONPB(%s): got:\n%v\n\nwant:\n%v", tt.desc, got, tt.want)
```

Comment:
TestUnmarshalWithJSONPB should already be printed by the golang test runner and can be omitted.

Reply:
done.